### PR TITLE
Update README for the mesh-reduced example

### PR DESCRIPTION
**Pull request by mnabian**
# Modulus Pull Request

## Description

Updates the documentation for the mesh-reduced example for how to download the dataset

## Checklist

- [ ] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [ ] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

File: examples/cfd/vortex_shedding_mesh_reduced/README.md [hunk 1]
```diff
@@ -59,11 +59,12 @@ per GPU is set to 10 for the sequence model training. Traing epochs is set as 20
...
 
 ```bash
-cd dataset
-sh TODO: need a shell to download the raw dataset from a cloud storage.
+wget --content-disposition https://api.ngc.nvidia.com/v2/resources/nvidia/modulus/modulus_datasets_cylinder-flow/versions/v1/zip -O modulus_datasets_cylinder-flow_v1.zip
+unzip modulus_datasets_cylinder-flow_v1.zip
+unzip dataset.zip
 ```
 
-This example requires the `torch-scatter` and  `torch-cluster` library for the
```
Comment (ktangsali) [inline examples/cfd/vortex_shedding_mesh_reduced/README.md:66]: Should be `torch-cluster` itself right?

**Review by ktangsali — APPROVED** [review]

File: examples/cfd/vortex_shedding_mesh_reduced/README.md (hunk 1 above)
Comment (mnabian) [reply examples/cfd/vortex_shedding_mesh_reduced/README.md:66]: Thanks, fixed it

**Pull request merged**